mmv1/third_party/terraform/website/docs/r/container_cluster.html.markdown

```diff
@@ -1064,6 +1064,8 @@ sole_tenant_config {
 
 * `enable_nested_virtualization`- (Optional) Defines whether the instance should have nested virtualization enabled. Defaults to false.
 
+* `performance_monitoring_unit` - (Optional) Defines the performance monitoring unit [PMU](https://cloud.google.com/compute/docs/pmu-overview) level. Defaults to off.
```

Comment:
nit: Would it make sense to also list currently available values here? Linking the docs, as you've done, is sufficient, but could save users a click-through.